Classes/Emogrifier.php

```diff
@@ -1077,6 +1078,14 @@ function ($matches) use (&$media) {
             $cssWithoutComments
         );
 
+	//Keep @imports
+	$imports = '';
+	preg_match_all('/^\\s*@import\\s[^;]+;/misU', $css, $importsMatches, PREG_PATTERN_ORDER);
+	if( !empty($importsMatches[0]) ){
```

Comment:
No spaces after the opening parenthesis or before the closing one, please. But there should be a space between the closing parenthesis and the opening curly brace.